lang/java/reef-tests/src/main/java/org/apache/reef/tests/fail/task/Driver.java

```diff
@@ -135,6 +135,13 @@ public void onNext(final RunningTask task) {
             + " not equal expected ID " + taskId);
       }
 
+      // In case of slow machines, it takes some time until task is ready.
+      try {
+        Thread.sleep(1000);
```

Comment:
Hi, Markus. Thank you. I will change the code in another ways. 

What I mean by that was
```
public final class DefaultClientCloseHandler implements EventHandler<Void> {
...
  public void onNext(final Void aVoid) {
```
is invoked instead of 
```
public final class FailTaskClose implements Task, EventHandler<CloseEvent> {
...
  public void onNext(final CloseEvent event) throws SimulatedTaskFailure {
```